compose.yml

```diff
@@ -1,12 +1,24 @@
 ---
 # This playbook composes OSTree
+
+- include_vars:
+    file: vars/packages.yml
+
 - name: Clone Fedora-Atomic buildscripts
   git:
     repo=https://pagure.io/fedora-atomic.git
     version=f25
     depth=1
     dest=/workspace/fedora-atomic
 
+- name: Customize Packages
+  lineinfile:
```

Comment:
You would not create a copy, but a child: https://gitlab.com/gbraad/ceph-atomic/blob/downstream/ceph-atomic-host.json The key would be to use: `"include": "centos-atomic-host.json",` to create the relationship.